### PR TITLE
Error update

### PR DESCRIPTION
**Pull request by AustinSeto**
Part of https://github.com/OpenLiberty/guides-common/issues/243

### CODE REVIEW

File: finish/src/main/java/io/openliberty/guides/inventory/InventoryResource.java [hunk 1]
```diff
@@ -81,8 +81,10 @@ public Response getPropertiesForHost(
         Properties props = manager.get(hostname);
         if (props == null) {
             return Response.status(Response.Status.NOT_FOUND)
-                            .entity("ERROR: Unknown hostname or the system service may "
-                                + "not be running on " + hostname)
+                            .entity("{ \"error\" : " 
+                                    + "\"Unknown hostname " + hostname 
+                                    + "or the resource may not be "
```
Comment (gkwan-ibm) [inline finish/src/main/java/io/openliberty/guides/inventory/InventoryResource.java:86]: Need a space before the "or" word as in the start directory

Comment (AustinSeto) [reply finish/src/main/java/io/openliberty/guides/inventory/InventoryResource.java]: Adjustment made

**Comment by TiagoF99** [comment]
Looks good.

**Comment by TiagoF99** [comment]
Verified on QA. Looks good.

**Pull request merged**